scripts/batchaipcreator.py

```diff
@@ -1,14 +1,11 @@
 #!/usr/bin/env python3
 '''
-Batch process packages by running accession.py and makepbcore.py
+Batch process packages by running aipcreator.py and makepbcore.py
 The outcome will be:
-* Packages are accessioned
+* Packages are aipped
```

Comment:
Again, aipped as a verb seems like it doesn’t work? It’s pretty much saying “packages are packaged”

Reply:
@hyz2993 - suggest changing all instances of 'packages are aipped' or 'package is aipped' to AIP is created.